server/src/infra/repositories/access.repository.ts

```diff
@@ -24,6 +24,7 @@ type ITimelineAccess = IAccessRepository['timeline'];
 type IPersonAccess = IAccessRepository['person'];
 type IPartnerAccess = IAccessRepository['partner'];
 
+@DecorateAll(ExecutionTimeHistogram())
```

Comment:
Do we want to make an alias for this? Something like
```ts
export const Instrumentation = () => DecorateAll(ExecutionTimeHistogram());
```

Reply:
That's a good idea